### PR TITLE
Reorganized league stats into game manager

### PR DESCRIPTION
**Pull request by cpfergus1**
Reorganized league stats into game_manager

### CODE REVIEW

**Review by GregJMitchell — APPROVED** [review]
Great job moving all the methods. Very readable and concise methods

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -14,12 +14,36 @@ def initialize(locations)
   end
 
   def load_managers(locations)
```
Comment (s-steel) [inline lib/stat_tracker.rb]: I'm interested to see if the ordering here matters after we merge everything.

File: lib/team_manager.rb [hunk 2]
```diff
@@ -2,10 +2,11 @@
 require 'csv'
 
 class TeamManager
-  attr_reader :teams
+  attr_reader :teams, :teams_data
```
Comment (s-steel) [inline lib/team_manager.rb:5]: This looks nice, it tightens up some methods.

**Review by s-steel — APPROVED** [review]
Looks good.  Glad to see some mocks and stubs being used.

**Review by c-bartell — APPROVED** [review]

**Pull request merged**